### PR TITLE
 Add Filen.app 2.0.4

### PR DESCRIPTION
**Pull request by Nebow**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/filen.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  version "2.0.4"
+  sha256 :no_check
+
+  url "https://cdn.filen.io/desktop/release/filen_#{arch}.dmg"
+  name "Filen"
+  desc "Desktop client for Filen.io"
+  homepage "https://filen.io/"
+
+  livecheck do
+    url "https://github.com/FilenCloudDienste/filen-desktop/releases/latest/"
```
Comment (bevanjkay) [inline Casks/filen.rb:13]: @Nebow Do you know if upstream use `pre-releases` or if they publish tags before releases?
If not, we can use the `:git` strategy instead, which is a more efficient strategy and should be used where possible.

**Review by bevanjkay — DISMISSED** [review]

**Comment by p-linnane** [comment]
Thanks for contributing to `homebrew-cask` @Nebow. Please do not use merge commits, as it causes issues with the merge process.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @Nebow.

**Pull request merged**